content/community/sig/index.md

```diff
@@ -0,0 +1,427 @@
+---
+title: Special Interest Groups (SIG)
+autotoc: false
+---
+
```

Comment:
every other section seems to have 'keywords' to describe the section - do you want to add some here or delete them everywhere else?
If keeping the format, then keywords might be:
'communities of practice, regional communities, scientific projects, training? (where has training gone?'

Reply:
I will add keywords.

I would love the GTN to become an official SIG. But that's just my opinion.